app/exceptions/NoEligibleCphError.js

```diff
@@ -0,0 +1,8 @@
+class NoEligibleCphError extends Error {
```

Comment:
I don't see where this is being thrown, still WIP?

Reply:
yeah exactly, there for a future task when we integrate the CPH check API